ui/app/pages/first-time-flow/seed-phrase/confirm-seed-phrase/confirm-seed-phrase.component.js

```diff
@@ -271,9 +271,5 @@ function insert (list, value, target, removeOld) {
     })
   }
 
-  if (nextList.length > 12) {
-    nextList.pop()
-  }
-
```

Comment:
I'm not really sure what this does, but it looks like it would be an issue if I left it behind.

Reply:
Yeah this can go